src/pages/products/sgid/boundaries/utah-state.astro

```diff
@@ -0,0 +1,97 @@
+---
+import Layout from '@layouts/DataPage.astro';
+
+import type { IMetadata, IPageMetadata } from '@models/products/sgid/types';
+import { ProductType, SgidCategory } from '@models/products/sgid/types';
+
+import HubDownloads from '@components/data/HubDownloads.astro';
+import Metadata from '@components/data/Metadata.astro';
+import UpdateHistory from '@components/data/UpdateHistory.astro';
+
+import CardWithPopularLink from '@components/page/CardWithPopularLink.astro';
+import CardWithSmallLink from '@components/page/CardWithSmallLink.astro';
+import GridForMoreResources from '@components/page/GridForMoreResources.astro';
+import GridForYouMightLike from '@components/page/GridForYouMightLike.astro';
+import Section from '@components/page/Section.astro';
+
+import { dataPages } from '@data/downloadMetadata';
+
+export const metadata: IMetadata = {
+  title: 'Utah State Boundary',
+  description: `Represent the boundary of the State of Utah for mapping purposes at a scale of 1:24000 or coarser.`,
+  stewards: ['UGRC', `Lieutenant Governor's Office`],
+  type: ProductType.POLYGON,
+  category: SgidCategory.BOUNDARIES,
+};
+
+const page: IPageMetadata = {
+  ...metadata,
+  updateHistory: [`July 2014`, `November 2011`, `March 2011`],
+
+  hub: {
+    ...dataPages[metadata.title],
+  },
+
+  // links are optional and only used when a product has download type of links that aren't ArcGIS Hub
+  // delete this if it is unused
+  links: [
+    {
+      title: 'The link title',
+      url: 'https://link-to-the-resource.com',
+    },
+  ],
+};
+---
+
+<Layout title={page.title} subTitle={page.updateHistory[0]} category={page.category} actionUrl={page.application}>
+  <Metadata slot="metadata" {...page} />
+
+  <Fragment slot="summary">
+    <p>
+      Utah State Boundary is maintained by UGRC with the help of many state, county, and local entities. Changes and
```

Comment:
**thought**: i thought this data was created by @ZachBeck  from esri data or some other existing data set and then we removed non bordering states.